app/views/facility_reservations/_reservation_fields.html.haml

```diff
@@ -1,4 +1,7 @@
 - if @reservation.admin_editable?
+  .control-group
```

Comment:
We need to be careful. This is ending up on the normal facility reservation edit page (accessed by the New/In-process tab under Reservations). We want it to only be there for admin reservations.